docs/resources/backup.md

```diff
@@ -0,0 +1,82 @@
+---
+page_title: "cloudavenue_backup Resource - cloudavenue"
+subcategory: "Backup"
+description: |-
+  The cloudavenue_backup resource allows you to manage backup from NetBackup solution.
+---
+
+# cloudavenue_backup (Resource)
+
+The `cloudavenue_backup` resource allows you to manage backup from NetBackup solution.
```

Comment:
Add more detail : Backup is available for VDC, VAPP or VM and add link to documentation